pyproject.toml

```diff
@@ -8,8 +8,13 @@ authors = [
 
 [tool.poetry.dependencies]
 python = "^3.6.2"
-rpaframework = { path = "packages/main", extras = ["cv", "playwright", "aws"] }
+rpaframework = { path = "packages/main" }
 rpaframework-google = { path = "packages/google" }
+rpaframework-recognition = { path = "packages/recognition" }
+rpaframework-windows = { path = "packages/windows" }
```

Comment:
Might pose issues when we get incompatibilities between **main**, **google**, **recognition** and **windows** taken from the latest master. (during common packages with different major versions)

Do we want to still pin version numbers here @mikahanninen ? (instead of direct source reference)